lib/pub_sub/message.rb

```diff
@@ -13,18 +13,17 @@ def initialize(payload)
     def process
       begin
         validate_message!
-      rescue PubSub::ServiceUnknown => e
-        return
+        handler.process(data)
```

Comment:
Not clear what's expected here? You'd like `validate_message!` to throw an error, but for `handler.process` not to?